app/assets/stylesheets/style.scss

```diff
@@ -263,22 +285,22 @@ label:hover {
 }
 
 @media only screen and (max-width: 40em) {
-    .title-area {
+  .title-area {
 
-    }
-    .name {
+  }
+  .name {
 
-    }
+  }
 
-    .name img {
-            height:35px;
-            margin-top:5px;
-            margin-left:5px;
-    }
+  .name img {
+    height:35px;
+    margin-top:5px;
```

Comment:
Properties should be ordered height, margin-left, margin-top<br>Colon after property should be followed by one space